src/witan/sen2/sen2.clj

```diff
@@ -0,0 +1,24 @@
+(ns witan.sen2.sen2
+  (:require [clojure.java.io :as io]
+            [tablecloth.api :as tc]))
+
+"A script which pulls data for a single gss-code from the SEN2 caseload, and generates total SEN2 counts per LAn"
+
+(def caseload-2010-2022 (delay (-> "sen2_estab_caseload.csv"
+                                   io/resource
+                                   io/file
+                                   tc/dataset))) ;; this doesn't work with districts
+
+(defn process-sen2-caseload [gss-or-la year caseload-data]
+  (-> caseload-data
+      (tc/select-rows #(and (or (= (get % "new_la_code") gss-or-la)
+                                (= (get % "la_name") gss-or-la))
+                            (= (get % "time_period") year)
+                            (= (get % "establishment_type") "Total")
+                            (= (get % "establishment_group") "Total")))
+      (tc/select-columns ["Total_all" "time_period"])))
+
+(defn generate-current-pop [gss]
+  (apply tc/concat
```

Comment:
would be good to pull this out with a single filter so that we don't have to concatenate the results. It is fine with data this small, just a bad habit really.

Reply:
Yes, I think this is actually just a hold-over from the previous version where this map was reading different files/dataset